project/assets/grid.less

```diff
@@ -0,0 +1,39 @@
+.grid {
+    width: 100%;
+    display: flex;
+    margin: 0 auto;
+}
+
+.col-1 {
+    flex: 0 0 33.33%;
+}
+
+.col-2 {
+    flex: 0 0 66.66%;
+}
+
+.col-3 {
+    flex: 0 0 100%;
+}
+
+.col-offset-2 {
+    margin-left: 66.66%;
+}
+
+@media (min-width: 768px) and (max-width: 1200px) {
```

Comment:
If this media query block is not going to change (too much), consider removing the `min-width` constraint and just keeping the `max-width` constraint. This is a rather common practice in responsive design (either desktop fist or mobile first, though the targeted constraints would be reversed): you specify common styling first, then override stuff incrementally, based on whatever responsive breakpoints you're targeting.